styles.css

```diff
@@ -416,3 +408,179 @@ textarea {
   text-align: center;
   padding-bottom: 45px;
 }
+
+@media screen and (min-width: 768px) {
+  .intro {
+    background-image: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    padding-right: 260px;
+    padding-left: 260px;
+    overflow: hidden;
+  }
+
+  .intro-text {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .intro h1 {
+    width: 50%;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    padding-bottom: 12px;
+  }
+
+  .intro p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .hamburger_menu {
+    display: none;
+  }
+
+  .desk {
+    display: flex;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 155px;
+    padding-right: 155px;
+  }
+
+  .page-container {
+    display: grid;
+    grid-template-columns: auto auto;
+    padding: 24px;
+    gap: 46px;
+  }
+
+  .work-section .page-container:nth-of-type(even) .img-wrap {
+    order: 2;
+  }
+
+  ul {
+    gap: 32px;
+    padding-top: 20px;
+  }
+
+  .img-wrap img {
+    height: 448px;
+  }
+
+  h2 {
+    font-size: 40px;
+    line-height: 52px;
+    padding-top: 28px;
+  }
+
+  .details p {
+    width: 100%;
+    padding-top: 20px;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .details .can {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .details .back {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .details .year {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .languages ul {
+    padding-bottom: 14px;
+  }
+
+  .work-section {
+    width: 100%;
+    padding-left: 142px;
+    padding-right: 142px;
+  }
+
+  .about-myself {
+    display: flex;
+    flex-direction: row;
+    gap: 37px;
+    align-items: baseline;
+    padding: 142px;
+  }
+
+  .about-myself h1 {
+    width: auto;
+  }
+
+  .asu,
+  .dropdown-items {
+    flex: 1 1;
+  }
+
+  .lightweight {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .title {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .dropdown-title,
+  .dropdown-title-open {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .vivo {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 122px;
+    align-items: flex-start;
+  }
+
+  .vivo h5 {
+    padding-left: 12px;
+  }
+
+  .footer-background {
+    display: flex;
+    align-items: center;
+    justify-items: center;
+  }
```

Comment:
- [ ] Hello @joshualyngdoh, I must commend that you have done wonderful work on your project and you made a great effort in getting the desktop version of the project done 👏 👏 🌹 . But I would kindly suggest we stick to the Figma template for the desktop version as much as possible. Currently, the `contact` section of your project is missing the `background-image` as we have it in the Figma template, Kindly see to fixing it before re-submitting for review

Here is the Figma design 👇 

![cont](https://user-images.githubusercontent.com/78634841/206574016-566f89ab-9835-44ca-9c2f-f4d13f5a3edd.png)

Your Design 👇 

![josh](https://user-images.githubusercontent.com/78634841/206574255-1ecdfdee-83c7-45d9-ab73-ec145d76ca2a.png)

- [ ] Kindly work on the `border-radius` around the `input` form too, to look perfect like what we have in the Figma design.

Reply:
Thank you. I will make all the changes and request a new review.